.stdci.yml

```diff
@@ -6,4 +6,4 @@ Architectures:
       runtime-requirements:
         host-distro: same
 script:
-    from-file: automation/run-tests.sh
+  from-file: automation/run-tests.sh --el8 --test-type integ
```

Comment:
Unfortunately you cannot pass arguments here. You can use env variables (see file in automation).